PocketKit/Sources/PocketGraph/user-defined-operations/unifiedHome.graphql

```diff
@@ -40,3 +44,7 @@ query HomeSlateLineup($locale: String!) {
         }
     }
 }
+
+fragment CollectionSummary on Collection {
+    slug
+}
```

Comment:
I like that you broke it into a fragment, shall we apply this `...CollectionSummary` fragment on `ItemParts` as well?

Reply:
unless we need more fields on `ItemParts`? If so, we can also create another fragment with a different name.